packages/backend-for-frontend/src/routers/catalogRouter.ts

```diff
@@ -968,18 +968,43 @@ const catalogRouter = (
       }
     })
     .post(
-      "/templates/eservices/:eServiceId/descriptors/:descriptorId/interface",
+      "/templates/eservices/:eServiceId/descriptors/:descriptorId/interface/soap",
       async (req, res) => {
         const ctx = fromBffAppContext(req.ctx, req.headers);
         try {
-          const response =
-            await eserviceTemplateService.addEserviceInterfaceByTemplate(
+          const eserviceId =
+            await eserviceTemplateService.addEserviceInterfaceSoapByTemplate(
```

Comment:
I would name the service the same as it is named the related `operationId` inside the `bffApi.yml`
```suggestion
            await eserviceTemplateService. addEServiceTemplateInstanceInterfaceSoap(
```